app/models/invite.js

```diff
@@ -13,33 +12,18 @@ export default Model.extend({
     updatedAtUTC: attr('moment-utc'),
     updatedBy: attr('number'),
     status: attr('string'),
-    roles: hasMany('role', {
-        embedded: 'always',
-        async: false
-    }),
+    role: belongsTo('role', {async: false}),
 
     ajax: injectService(),
     ghostPaths: injectService(),
 
-    role: computed('roles', {
-        get() {
-            return this.get('roles.firstObject');
-        },
-        set(key, value) {
-            // Only one role per user, so remove any old data.
-            this.get('roles').clear();
-            this.get('roles').pushObject(value);
-
-            return value;
-        }
-    }),
-
     resend() {
         let fullInviteData = this.toJSON();
         let inviteData = {
             email: fullInviteData.email,
-            roles: fullInviteData.roles
+            role: fullInviteData.role
```

Comment:
As far as i understood Ember, this get's called:
https://github.com/TryGhost/Ghost-Admin/pull/394/files#diff-4988c6daa641a939ebddb019fbf5d3dcR18

That's why no need to use role_id?